src/views/PropTableView.vue

```diff
@@ -441,32 +442,51 @@ onBeforeMount(() => {
     }
 });
 
+function ensureProfiles() {
+    return new Promise<void>((resolve, reject) => {
+        let expTimer = setTimeout(reject, 10 * 1000); // time out after 10s
+               
+        (function waitForProfiles() {
+            if (Object.keys(ui.profiles).length > 0) {
+                clearTimeout(expTimer)
+                return resolve()
+            };
+            setTimeout(waitForProfiles, 500);
+        })();
+    })
+}
```

Comment:
Not a big deal but, possible to move to `src/util`?

Reply:
It is defined in `ItemListView.vue` as well.

Reply:
Done in latest commit